ui/components/multichain/account-overview/account-overview-btc.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { BtcOverview } from '../../app/wallet-overview';
+import { AccountOverviewLayout } from './account-overview-layout';
+import { AccountOverviewCommonProps } from './common';
+
+export type AccountOverviewBtcProps = AccountOverviewCommonProps;
+
+export const AccountOverviewBtc = (props: AccountOverviewBtcProps) => {
+  return (
+    <AccountOverviewLayout
+      showTokens={true}
+      showTokensLinks={false}
+      showNfts={false}
+      showActivity={true}
+      {...props}
+    >
+      {
+        ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask,build-mmi)
+        <BtcOverview />
+        ///: END:ONLY_INCLUDE_IF
```

Comment:
What is the point of this code fence?

Reply:
Just used the same pattern than for `account-overview-eth`, but that's maybe legacy code fences here 😅 Do you want me to remove those?

Reply:
Then, we'd better keep it like this for now, IMO